### ISSUE TITLE
can not install

### ISSUE THREAD

**Issue by lighthousebulb** [agsh/onvif#167]
As soon as I try to install `npm install agsh/onvif` it failes with the following log

Any ideas?

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'install', 'agsh/onvif' ]
2 info using npm@6.14.5
3 info using node@v12.18.1
4 verbose npm-session 4de735fa2d1c2431
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 silly fetchPackageMetaData error for github:agsh/onvif Error while executing:
7 silly fetchPackageMetaData undefined ls-remote -h -t ssh://git@github.com/agsh/onvif.git
7 silly fetchPackageMetaData
7 silly fetchPackageMetaData
7 silly fetchPackageMetaData spawn git ENOENT
8 timing stage:rollbackFailedOptional Completed in 1ms
9 timing stage:runTopLevelLifecycles Completed in 218ms
10 verbose stack Error: spawn git ENOENT
10 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
10 verbose stack     at onErrorNT (internal/child_process.js:469:16)
10 verbose stack     at processTicksAndRejections (internal/process/task_queues.js:84:21)
11 verbose cwd /home/xxx
12 verbose Linux 5.4.41-1-pve
13 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "agsh/onvif"
14 verbose node v12.18.1
15 verbose npm  v6.14.5
16 error code ENOENT
17 error syscall spawn git
18 error path git
19 error errno ENOENT
20 error enoent Error while executing:
20 error enoent undefined ls-remote -h -t ssh://git@github.com/agsh/onvif.git
20 error enoent
20 error enoent
20 error enoent spawn git ENOENT
21 error enoent This is related to npm not being able to find a file.
22 verbose exit [ 1, true ]
```

**Comment by chriswiggins** [comment]
Do you have the git client installed? It looks like it’s trying to find git and can’t. 

Why don’t you `npm install onvif`

**Comment by lighthousebulb** [comment]
I tried to install another package, that required "agsh/onvif".
Looks like the package [node-red-contrib-onvif-nodes](https://github.com/bartbutenaers/node-red-contrib-onvif-nodes) requires "agsh/onvif" because of some extra features? Just with "onvif" the install procedure was not satisfied.

I tried to install `npm install onvif` - it worked fine, but the mentioned package is requesting the "latest" Version and seems not to be satisfied with the "stable" version of onvif.

Well, the README of onfiv suggests to use `npm install agsh/onvif` to get the latest version... and that does not work...

I tracked the install procedure down to fail with "agsh/onvif", what is why I opened the issue here and not in the "node-red-contrib-onvif-nodes"
